---
[v1] arm64: dts: mediatek: add mt7981 pdma interrupts

Add additional PDMA interrupt lines (189-192) to ethernet controller for MT7981
to support RSS (Receive Side Scaling) and HWLRO (Hardware Large Receive Offload)
features for improved network performance.
---
v1: Move openwrt-master files to openwrt-25.12 branch after OpenWrt official
    25.12 branch release.
---

From de71a18c49c1ac4eb95efb3aa7f7cd6313eb364b Mon Sep 17 00:00:00 2001
From: Mason Chang <mason-cw.chang@mediatek.com>
Date: Fri, 5 Dec 2025 13:30:40 +0800
Subject: [PATCH] dts: mediatek: add mt7981 pdma interrupts for rss and hwlro

---
 arch/arm64/boot/dts/mediatek/mt7981b.dtsi | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

--- a/arch/arm64/boot/dts/mediatek/mt7981b.dtsi
+++ b/arch/arm64/boot/dts/mediatek/mt7981b.dtsi
@@ -442,7 +442,13 @@
 			interrupts = <GIC_SPI 196 IRQ_TYPE_LEVEL_HIGH>,
 				     <GIC_SPI 197 IRQ_TYPE_LEVEL_HIGH>,
 				     <GIC_SPI 198 IRQ_TYPE_LEVEL_HIGH>,
-				     <GIC_SPI 199 IRQ_TYPE_LEVEL_HIGH>;
+				     <GIC_SPI 199 IRQ_TYPE_LEVEL_HIGH>,
+				     <GIC_SPI 189 IRQ_TYPE_LEVEL_HIGH>,
+				     <GIC_SPI 190 IRQ_TYPE_LEVEL_HIGH>,
+				     <GIC_SPI 191 IRQ_TYPE_LEVEL_HIGH>,
+				     <GIC_SPI 192 IRQ_TYPE_LEVEL_HIGH>;
+			interrupt-names = "fe0", "fe1", "fe2", "fe3",
+					"pdma0", "pdma1", "pdma2", "pdma3";
 			clocks = <&ethsys CLK_ETH_FE_EN>,
 				<&ethsys CLK_ETH_GP2_EN>,
 				<&ethsys CLK_ETH_GP1_EN>,
